Add new objects

Muzzarino opened the pull request.

Muzzarino committed:
Revise cage scaling

Muzzarino committed:
Add few new objects

Muzzarino committed:
Create Object1182_Thunder

Muzzarino committed:
Create Object118B_Grass

Muzzarino committed:
Create Object118D_Mask

Muzzarino committed:
Add new objects

dreamsyntax commented on HeroesPowerPlant/LayoutEditor/SetObjects/ObjectClassesHeroes/List11/Object118D_Mask:
Missing .cs extension,
Remove unused imports.
Otherwise looks good to me.

dreamsyntax commented on HeroesPowerPlant/LayoutEditor/SetObjects/ObjectClassesHeroes/List11/Object1182_Thunder:
Missing .cs extension,
Remove unused imports.
Otherwise looks good to me.

Muzzarino committed:
Rename Object1182_Thunder to Object1182_Thunder.cs

Muzzarino committed:
Rename Object118B_Grass to Object118B_Grass.cs

Muzzarino committed:
Rename Object118D_Mask to Object118D_Mask.cs

Muzzarino committed:
New glow version 1.01

Muzzarino committed:
Re-add switch properties

Now that the glow effect for the switch matches the game, they can finally be rendered on scene edit.
